code/client/clrcore/External/VehicleHash.cs

```diff
@@ -30,24 +47,24 @@ public enum VehicleHash : uint
 		Baller4 = 634118882u,
 		Baller5 = 470404958u,
 		Baller6 = 666166960u,
+		Baller7 = 359875117u,
 		Banshee = 3253274834u,
 		Banshee2 = 633712403u,
 		Barracks = 3471458123u,
 		Barracks2 = 1074326203u,
 		Barracks3 = 630371791u,
+		Barrage = 4081974053u,
 		Bati = 4180675781u,
 		Bati2 = 3403504941u,
 		Benson = 2053223216u,
 		Besra = 1824333165u,
 		BestiaGTS = 1274868363u,
-		BF400 = 86520421u,
```

Comment:
Uhm, why add this one above unordered and remove this one which seems nicely ordered